command/command.go

```diff
@@ -175,70 +177,92 @@ func (w *backgroundCmd) Start() error {
 // KeepAlive will create a goroutine to send regular SIGCONT signal to associated command
 // a single goroutine will be launched no matter how many calls are done to KeepAlive
 func (w *backgroundCmd) KeepAlive() {
+	w.Lock()
+	defer w.Unlock()
+
 	if w.DryRun() {
 		return
 	}
 
-	w.Lock()
-	defer w.Unlock()
-
 	if w.ticker != nil {
 		return
 	}
 
 	w.ticker = time.NewTicker(cmdKeepAliveTickDuration)
 
+	w.chKeepAliveQuit = make(chan int)
```

Comment:
nit: I dont think we need to prefix this with `ch` just to show its type in the name